src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -113,16 +119,16 @@ class HaConfigDashboard extends LitElement {
 
   @property() public cloudStatus?: CloudStatus;
 
-  // null means not available
-  @property() public supervisorUpdates?: SupervisorAvailableUpdates[] | null;
-
   @property() public showAdvanced!: boolean;
 
   @state() private _tip?: string;
 
   private _notifyUpdates = false;
 
   protected render(): TemplateResult {
+    const canInstallUpdates = updateEntities(this.hass.states).filter(
+      (entity) => updateCanInstall(entity)
```

Comment:
Why not do this filtering inside the memoized call ? 

Should that filter also sort it by state name ?

Reply:
It's better to hit a memoized call here and do an extra filter in the `_handleAction`, because when handling an action there is nothing else going on. We want to do as little as possible when rendering.